src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  transition: transform $transition-time;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
```

Comment:
https://codeguide.academy/html-css.html#css-order

Please fix this in all places in code

Reply:
@Moroz-Dmytro used this one https://9elements.com/css-rule-order/ 
received it at one of the previous review